packages/astro/src/@types/astro.ts

```diff
@@ -1902,6 +1989,11 @@ export type AstroFeatureMap = {
 	 * The adapter can emit static assets
 	 */
 	assets?: AstroAssetsFeature;
+
+	/**
+	 * List of features that orbit around the i18n routing
+	 */
+	i18n?: AstroInternationalisationFeature;
```

Comment:
```suggestion
	i18n?: AstroInternationalizationFeature;
```

🇺🇸 Looks like we're also using `z` in the docs